context/context.go

```diff
@@ -82,6 +83,7 @@ type PccPolicy struct {
 type PcfSubscriberPolicyData struct {
 	Supi      string
 	PccPolicy map[string]*PccPolicy // sst+sd is key
+	CtxLog    *logrus.Entry
```

Comment:
You can add few fixed fields, like key fields to identify that ctxt uniquely as part of ctxt log

Reply:
sure. i will take this comment in my next commit